src/entities/projectVerificationForm.ts

```diff
@@ -74,13 +74,16 @@ export class ProjectContacts {
 export class Milestones {
   @Field(type => String, { nullable: true })
   foundationDate?: String;
-
   @Field({ nullable: true })
   mission?: string;
   @Field({ nullable: true })
   achievedMilestones?: string;
   @Field(type => [String], { nullable: true })
   achievedMilestonesProofs?: string[];
+  @Field(type => String, { nullable: true })
+  problem?: string;
+  @Field(type => String, { nullable: true })
```

Comment:
I tested adding the new fields with existing data.
Then removing then with existing that had this new fields

and the info was not deleted. This doesn't seem to delete data when server run db sync. So its safe